products/llbuildSwift/BuildSystemBindings.swift

```diff
@@ -368,27 +350,6 @@ public struct CommandExtendedResult {
 
 }
 
-/// The BuildKey encodes the key space used by the BuildSystem when using the
-/// core BuildEngine.
-public struct BuildKey {
-    public typealias Kind = BuildKeyKind
-
-    /// The kind of key
-    public let kind: Kind
-
-    /// The actual key data
-    public let key: String
-
-    public init(kind: Kind, key: String) {
```

Comment:
Should I try to maintain this API by giving the possibility to create plain `BuildKey`s?